tests/Functional/Admin/SharedBlockAdminTest.php

```diff
@@ -45,11 +45,151 @@ public function testCrudUrls(string $url, array $parameters = []): void
+        ]];
+
+        yield 'Create Shared Block - Template' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'uniqid' => 'shared_block',
+            'type' => 'sonata.block.service.template',
+        ], 'btn_create_and_list', [
+            'shared_block[name]' => 'Name',
+            'shared_block[enabled]' => 1,
+            'shared_block[settings][template]' => '@SonataBlock/Block/block_template.html.twig',
+        ]];
+
+        yield 'Create Shared Block - Menu' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'uniqid' => 'shared_block',
+            'type' => 'sonata.block.service.menu',
+        ], 'btn_create_and_list', [
+            'shared_block[name]' => 'Name',
+            'shared_block[enabled]' => 1,
+            'shared_block[settings][title]' => 'Title',
+            'shared_block[settings][menu_name]' => 'sonata_admin_sidebar',
+            'shared_block[settings][safe_labels]' => 1,
+            'shared_block[settings][current_class]' => 'active',
+            'shared_block[settings][first_class]' => 'first',
+            'shared_block[settings][last_class]' => 'last',
+            'shared_block[settings][menu_class]' => 'menu_sonata',
+            'shared_block[settings][children_class]' => 'children',
+            'shared_block[settings][menu_template]' => '@SonataBlock/Block/block_core_menu.html.twig',
+        ]];
+
+        yield 'Create Shared Block - RSS' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'uniqid' => 'shared_block',
+            'type' => 'sonata.block.service.rss',
+        ], 'btn_create_and_list', [
+            'shared_block[name]' => 'Name',
+            'shared_block[enabled]' => 1,
+            'shared_block[settings][url]' => 'https://sonata-project.org',
+            'shared_block[settings][title]' => 'Title',
+            'shared_block[settings][translation_domain]' => 'SonataPageBundle',
+            'shared_block[settings][icon]' => 'fa fa-home',
+            'shared_block[settings][class]' => 'custom_class',
+        ]];
+
+        // Blocks From SonataPageBundle
+        yield 'Create Shared Block - Children Pages' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'uniqid' => 'shared_block',
+            'type' => 'sonata.page.block.children_pages',
+        ], 'btn_create_and_list', [
+            'shared_block[name]' => 'Name',
+            'shared_block[enabled]' => 1,
+            'shared_block[settings][title]' => 'Title',
+            'shared_block[settings][translation_domain]' => 'SonataPageBundle',
+            'shared_block[settings][icon]' => 'fa fa-home',
+            'shared_block[settings][current]' => 1,
+            // 'shared_block[settings][pageId]' => 1,
+            'shared_block[settings][class]' => 'custom_class',
+        ]];
+
+        // yield 'Create Shared Block - Breadcrumb' => ['/admin/tests/app/sonatapageblock/shared/create', [
```

Comment:
This one is broken on SonataSeoBundle.

Reply:
Can it be fix ?

Reply:
This one is a bit problematic... Take a look at the class:

2.x: https://github.com/sonata-project/SonataSeoBundle/blob/2.x/src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php#L29

Here the class extends MenuBlockService which implements EditableBlockService 👍 

3.x: https://github.com/sonata-project/SonataSeoBundle/blob/3.x/src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php#L29

Here the class does not extend MenuBlockService, and does not implement EditableBlockService 👎 

To fix this we will need to BC break, but it should be accepted IMO, because that block is completely unusable.

Reply:
Maybe @eerison or other users of this bundle could help us with it. IMHO we should not block this PR for it, because it will need some thinking, we can create an issue or something.

Reply:
> To fix this we will need to BC break, but it should be accepted IMO, because that block is completely unusable.

Yeah, I agree

> we can create an issue or something.

Can you create an issue with the list of things to fix then ? Then we'll merge this one.

Reply:
> Maybe @eerison or other users of this bundle could help us with it. IMHO we should not block this PR for it, because it will need some thinking, we can create an issue or something.

Create an issue to check it's fine!

But this `breadcrumb menu`, is a bit trick! I'm still struggle with this in #1497  😢

Reply:
One for this: https://github.com/sonata-project/SonataSeoBundle/issues/685